You can pass an `Error` object to `captureException()` to get it captured as event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stacktrace.

```javascript
import * as Sentry from "@sentry/ember";

try {
  aFunctionThatMightFail();
} catch (err) {
  Sentry.captureException(err);
}
```
